test/compliment/sources/t_class_members.clj

```diff
@@ -35,60 +83,61 @@
     (strip-tags (src/members-candidates ".sta" (-ns) nil))
     => (just [".start" ".startsWith"] :in-any-order)
 
-    (strip-tags (src/members-candidates ".sta" (-ns) (ctx/parse-context
-                                                      '(__prefix__ ^String foo))))
+    (strip-tags (src/members-candidates ".sta" (-ns) (ctx/cache-context
+                                                      "(__prefix__ ^String foo)")))
     => (just [".startsWith"])
 
-    (strip-tags (src/members-candidates ".in" (-ns) (ctx/parse-context
-                                                     '(__prefix__ ^Thread foo))))
+    (strip-tags (src/members-candidates ".in" (-ns) (ctx/cache-context
+                                                     "(__prefix__ ^Thread foo)")))
     => (just [".interrupt"])
 
-    ;; read-string is used here because fudje seems to mess up metadata on forms
-    (strip-tags (src/members-candidates ".m" (-ns) (ctx/parse-context
-                                                     (read-string
-                                                      "(__prefix__ ^java.io.File (foo))"))))
+    (strip-tags (src/members-candidates ".m" (-ns) (ctx/cache-context
+                                                    "(__prefix__ ^java.io.File (foo))")))
     => (just [".mkdirs" ".mkdir"] :in-any-order)
 
-    (strip-tags (src/members-candidates ".in" (-ns) (ctx/parse-context
-                                                     (read-string
-                                                      "(__prefix__ ^Thread (foo))"))))
+    (strip-tags (src/members-candidates ".in" (-ns) (ctx/cache-context
+                                                     "(__prefix__ ^Thread (foo))")))
     => (just [".interrupt"])
 
-    (strip-tags (src/members-candidates ".p" (-ns) (ctx/parse-context
-                                                    '(__prefix__ ^java.util.Map foo))))
+    (strip-tags (src/members-candidates ".p" (-ns) (ctx/cache-context
+                                                    "(__prefix__ ^java.util.Map foo)")))
     => (just [".putAll" ".putIfAbsent" ".put"] :in-any-order)
 
-    (strip-tags (src/members-candidates ".in" (-ns) (ctx/parse-context
-                                                     '(let [^Thread foo ...
-                                                            ... ...
-                                                            _ (.start foo)]
-                                                        (__prefix__ foo)))))
+    (strip-tags (src/members-candidates ".in" (-ns) (ctx/cache-context
```

Comment:
Saw your other comment. Yeah, passing strings is more realistic since this is what clients do. But keeping forms in the tests is easier to manage (edit, indent).

I forgot about the problem with metadata in Fudje, gotta take a look.